examples/showcase-shopping/index.html

```diff
@@ -8,50 +8,47 @@
   </head>
   <body>
     <a-scene>
```

Comment:
[http://localhost:9000/examples/showcase-shopping/](http://localhost:9000/examples/showcase-shopping/)

this scene loads _much_ faster now. great work :+1:

<hr>

in Firefox, however, the sizes of the images appear off:

### [http://localhost:9000/examples/showcase-shopping/](http://localhost:9000/examples/showcase-shopping/)
<img width="1363" alt="screenshot 2016-02-17 03 34 40" src="https://cloud.githubusercontent.com/assets/203725/13108390/7c91753e-d527-11e5-9bc5-1fd11f58f43b.png">

### https://aframe.io/aframe/examples/showcase-shopping/
<img width="1363" alt="showcase_curved_mockups" src="https://cloud.githubusercontent.com/assets/203725/13108389/7c90436c-d527-11e5-968b-8aa920961757.png">

<hr>

and also in this branch compare in Chrome vs. Firefox:

![screenshot 2016-02-17 03 32 51](https://cloud.githubusercontent.com/assets/203725/13108416/a08a7db4-d527-11e5-9283-5f08cc9afd4b.png)
<img width="1363" alt="screenshot 2016-02-17 03 33 02" src="https://cloud.githubusercontent.com/assets/203725/13108417/a08fd728-d527-11e5-8708-ab8b4bd410ee.png">